try:
    pass
except ValueError:
    try:
        pass
    except OSError:
        pass
